workflows/model_spec.py

```diff
@@ -755,6 +755,9 @@ def expand_to_specs(self) -> List["ModelSpec"]:
                 max_concurrency=32,
                 max_context=128 * 1024,
                 default_impl=True,
```

Comment:
Actually, we should only have one default implementation for Qwen2.5-72B-* on T3K, so can we make this False
```suggestion
                default_impl=False,
```